IPyth.sol

```diff
@@ -7,8 +7,29 @@ import "./PythStructs.sol";
 /// @dev Please refer to the guidance at https://docs.pyth.network/consumers/best-practices for how to consume prices safely.
 /// @author Pyth Data Association
 interface IPyth {
-    /// @dev Emitted when price feed with `id` is updated.
-    event PriceUpdate(bytes32 indexed id, uint64 publishTime);
+    /// @dev Emitted when an update for price feed with `id` is processed successfully.
+    /// @param id The Pyth Price Feed ID.
+    /// @param stored It's true if the price update is more recent and stored.
+    /// @param chainId ID of the source chain that the batch price update containing this price.
+    /// @param sequenceNumber Sequence number of the batch price update containing this price.
+    /// @param existingPublishTime Publish time of the previously stored price.
```

Comment:
It might be nicer to change this to be the time since the last update on this price feed.

Reply:
Time since last update is a bit different here (last update != last publish time). You mean last publish time, yeah?
I agree it's more intuitive. What shall I name it? `timeSinceLastPublishTime` that is `int64`?

Reply:
`lastPublishTime`?

Reply:
I'm not sure it's good for diff 🤔

Reply:
I think they are equal information-wise. Your reasoning sounds good, so I'll keep lastPublishTime.